ast/src/analyzed/mod.rs

```diff
@@ -23,7 +23,7 @@ pub enum StatementIdentifier {
     Identity(usize),
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Now do we still need this? :)
We can keep it if you want.

Reply:
Nope, will remove :)